content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -10,45 +10,160 @@ content_template: templates/concept
+We will refer to these as "Kubernetes CA certificate and key".
+
+All Kubernetes components that use these certificates - kubelet, kube-apiserver, kube-controller-manager - assume the key and certificate to be PEM-encoded.
+
 ## kube-apiserver configuration
+The kube-apiserver has several requirements to enable TLS bootstrapping:
 
-The API server should be configured with an
-[authenticator](/docs/reference/access-authn-authz/authentication/) that can
-authenticate tokens as a user in the `system:bootstrappers` group.
+* Recognizing CA that signs the client certificate 
+* Authenticating the bootstrapping kubelet to the `system:bootstrappers` group
+* Authorize the bootstrapping kubelet to create a certificate signing request (CSR)
 
-This group will later be used in the controller-manager configuration to scope
-approvals in the default approval controller. As this feature matures, you
-should ensure tokens are bound to a Role Based Access Control (RBAC) policy
-which limits requests (using the [bootstrap
-token](/docs/reference/access-authn-authz/bootstrap-tokens/)) strictly to client
-requests related to certificate provisioning. With RBAC in place, scoping the
-tokens to a group allows for great flexibility (e.g. you could disable a
-particular bootstrap group's access when you are done provisioning the nodes).
+### Recognizing client certificates
+This is normal for all client certificate authentication.
+If not already set, add the `--client-ca-file=FILENAME` flag to the kube-apiserver command to enable
+client certificate authentication, referencing a certificate authority bundle
+containing the signing certificate, for example
+`--client-ca-file=/var/lib/kubernetes/ca.pem`.
+
+### Initial bootstrap authentication
+In order for the bootstrapping kubelet to connect to kube-apiserver and request a certificate, it must first authenticate to the server.
+You can use any [authenticator](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) that can authenticate the kubelet.
 
 While any authentication strategy can be used for the kubelet's initial
 bootstrap credentials, the following two authenticators are recommended for ease
 of provisioning.
 
-1. [Bootstrap Tokens](/docs/reference/access-authn-authz/bootstrap-tokens/) - __beta__
+1. [Bootstrap Tokens](#bootstrap-tokens) - __beta__
 2. [Token authentication file](#token-authentication-file)
 
-Using bootstrap tokens is currently __beta__ and will simplify the management of
-bootstrap token management especially in a HA scenario.
+Bootstrap tokens are a simpler and more easily managed method to authenticate kubelets, and do not require any additional flags when starting kube-apiserver. 
+Using bootstrap tokens is currently __beta__ as of Kubernetes version 1.12.
+
+Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user in the `system:bootstrappers` group. This group
```

Comment:
We can do that. I get what you are saying, but I do think this makes it  far easier to understand. Rather than, "you must set up a role with the right permissions and a group and... ", it is easier for people to understand, "make sure your user authenticates as `system:bootstrappers` and this all will work."

Maybe something like:

```
Whichever method you choose, the requirement is that the kubelet be able to authenticate as a user who has the right to create and get CSRs. You can create such a group or user on your own. In the interest of standardization and simplicity, Kubernetes ships with bootstrap tokens authenticating to the `system:bootstrappers` group, which we strongly recommend you simply adopt.
```